src/lib.rs

```diff
@@ -96,5 +96,8 @@ impl Plugin for ParticleSystemPlugin {
             .add_system(particle_color)
             .add_system(particle_transform)
             .add_system(particle_cleanup);
+
+        #[cfg(feature = "bevy_reflect")]
+        app.register_type::<ParticleSystem>();
```

Comment:
Should we add `core::ops::Range<f32>` here as well, so that it displays correctly in the inspector by default?

Reply:
Does it? I kind of assumed that was something that would have to be added to bevy-inspector-egui.

Reply:
Ah yes, I think you might be correct in that. Nevermind!